frontend/src/pages/ArtifactList.tsx

```diff
@@ -184,7 +192,72 @@ class ArtifactList extends Page<{}, PipelineListState> {
         } else {
           return compare ? 1 : -1;
         }
+      }));
+  }
+
+  /**
+   * Groups the incoming rows by name and type pushing all but the first row
+   * of each group to the expandedRows Map.
+   * @param rows
+   */
+  private groupRows(rows: Row[]): Row[] {
+    const flattenedRows = rows.reduce((map, r) => {
+      const stringKey = `${r.otherFields[2]}/${r.otherFields[0]}`;
```

Comment:
A comment here indicating what's at these indices would be helpful

Reply:
Thanks done!